src/protocols/rdp/guac_rdpdr/rdpdr_fs_service.h

```diff
@@ -42,7 +42,7 @@
  * Registers a new filesystem device within the RDPDR plugin. This must be done
  * before RDPDR connection finishes.
  */
-void guac_rdpdr_register_fs(guac_rdpdrPlugin* rdpdr);
+void guac_rdpdr_register_fs(guac_rdpdrPlugin* rdpdr, char* drive_name);
```

Comment:
The documentation for this function should be brought up-to-date now that it's being updated. Other than that, I think these changes are pretty much good to go.

Reply:
Updated.